README.md

```diff
@@ -5,14 +5,14 @@
 This ingress controller runs inside a Kubernetes cluster monitoring changes to the Ingress resources and orchestrating 
 AWS Load Balancers accordingly.
 
-It should be deployed once per Kubernetes cluster. It will use the EC2 instance metadata to find the CloudFormation
-stack it belongs to. This value is then used to discover the required AWS resources that are attached to each newly 
-created Application Load Balancer.
+It will use the EC2 instance metadata to find the CloudFormation
+stack it belongs to which is then used to discover the required AWS resources that are attached to each newly created Application Load Balancer.
 
 ## How it works
 
-It continuously polls the API server checking for ingress resources. For each cycle it creates load balancers for new
+The controller continuously polls the API server checking for ingress resources. For each cycle it creates load balancers for new
```

Comment:
This part is about how the kubernetes part works, should we also write how the aws part works?
We find load balancers by certificate and have ownership by tag. This is maybe something the user does not expect.